pom.xml

```diff
@@ -126,18 +125,19 @@
             <dependency>
                 <groupId>com.github.seratch</groupId>
                 <artifactId>jslack</artifactId>
-                <version>${jslack.version}</version>
+                <version>1.5.6</version>
```

Comment:
If w ejave declared jslcak dependency in parent pom then why its needed to define the same version in cild pom?

Reply:
Due to historical reasons Bitbucket Slack `pom.xml` isn't child pom to this parent `pom.xml` in the repository root. It causes duplicate definition of some dependencies.